### PR TITLE
Added OpenAI service

### PR DESCRIPTION
**Pull request by YevhenPoliakov**

### CODE REVIEW

File: WfpBotConsole/Services/OpenAI/OpenAIService.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using OpenAI_API;
+
+namespace WfpBotConsole.Services.OpenAI;
+
+public class OpenAIService : IOpenAIService
+{
+    private readonly OpenAIAPI _openAIAPI;
```
Comment (PHermanov) [inline WfpBotConsole/Services/OpenAI/OpenAIService.cs:10]: By conventions, acronims are still should follow the camel-case rules, like OpenAiApi, OpenAiService, etc

File: WfpBotConsole/WfpBotConsole.csproj [hunk 2]
```diff
@@ -47,4 +51,7 @@
 	  </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Services\OpenAI\" />
```
Comment (PHermanov) [inline WfpBotConsole/WfpBotConsole.csproj:55]: What is this for?

**Review by PHermanov — CHANGES_REQUESTED** [review]

File: WfpBotConsole/WfpBotConsole.csproj (hunk 2 above)
Comment (YevhenPoliakov) [reply WfpBotConsole/WfpBotConsole.csproj]: Fixed

File: WfpBotConsole/Services/OpenAI/OpenAIService.cs (hunk 1 above)
Comment (YevhenPoliakov) [reply WfpBotConsole/Services/OpenAI/OpenAIService.cs]: Renamed to "_openAiApi"

**Pull request merged**